react/components/ProductResults.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC } from 'react'
+import ProductSummary from 'vtex.product-summary/ProductSummaryCustom'
+import { Link } from 'vtex.render-runtime'
+import { useCssHandles } from 'vtex.css-handles'
+import { highlightTerm } from './utils'
+import HorizontalRule from './HorizontalRule'
+import { ISearchProduct } from '../models/search-product'
+
+interface ProductResultsProps {
+  products: ISearchProduct[]
+
+  onProductClick: (product: string, position: number) => void
+  inputValue: string
+}
+
+const CSS_HANDLES = [
+  'productResultsWrapper',
+  'productResultImage',
+  'productResultName',
+  'productResultLink',
+  'productResultHighlightedTerm',
+] as const
+
+const ProductResults: FC<ProductResultsProps> = ({
+  products,
+  onProductClick,
+  inputValue,
+}) => {
+  const { handles } = useCssHandles(CSS_HANDLES)
+
+  if (!products.length) return null
+
+  const productSummaryItems: Product[] = products.map(product =>
+    ProductSummary.mapCatalogProductToProductSummary(
+      product,
+      'FIRST_AVAILABLE',
```

Comment:
Is this `SKIP` vs `DEFAULT` or different?

Reply:
skip vs default happens at the API layer not the at the view layer: will tag you where that is now